stackslib/src/lib.rs

```diff
@@ -82,11 +82,9 @@ const BUILD_TYPE: &'static str = "release";
 
 pub fn version_string(pkg_name: &str, pkg_version: Option<&str>) -> String {
     let pkg_version = pkg_version.unwrap_or(STACKS_NODE_VERSION);
-    let git_branch = GIT_BRANCH
-        .map(|x| format!("{}", x))
-        .unwrap_or("".to_string());
-    let git_commit = GIT_COMMIT.unwrap_or("");
-    let git_tree_clean = GIT_TREE_CLEAN.unwrap_or("");
+    let git_branch = GIT_BRANCH_ENV.unwrap_or(GIT_BRANCH.unwrap_or(""));
```

Comment:
Since `unwrap_or()` is eagerly evaluated, it's best to only use it with literal values. Use `unwrap_or_else()` for anything non-trivial:

```suggestion
    let git_branch = GIT_BRANCH_ENV.unwrap_or_else(|| GIT_BRANCH.unwrap_or(""));
```